src/scenes/home/jobs/jobs.js

```diff
@@ -29,11 +32,39 @@ class Jobs extends Component {
     };
     tryRunInit();
   }
+
+  featuredJobs
   render() {
+    const featuredJobs = FeaturedJobsData
+      .filter(x => x.status === 'active')
+      .map(job => (
+        <FeaturedJob
+          key={`${Math.random()} + ${job.name} + ${job.sourceUrl}`}
+          title={job.title}
+          source={job.source}
+          sourceUrl={job.sourceUrl}
+          city={job.city}
+          state={job.state}
+          country={job.country}
+          description={job.description}
+          status={job.status}
+          remote={job.remote}
+        />
+      ));
     return (
-      <Section title="Open Positions" theme="white">
-        <div id="zipsearch_container" />
-      </Section>
+      <div>
+        <Section title="Featured Jobs" theme="white">
+          <div className={styles.featuredJobsContainer}>
+            <div className={styles.featuredJobs}>
+              {featuredJobs}
```

Comment:
You should do

```
{FeaturedJobsData.length() > 0 ? featuredJobs : <div>some default content expressed a lack of featured jobs and how to get one featured</div>}
```

Reply:
Good idea. We should probably have a contact link somewhere here.  It should display whether or not there are any featured jobs exist. Can start off with an email link to @ksmacleod99 for now.

Reply:
@kylemh As long as we have the contact link (explaining how to get a featured job), we can probably go live with this sooner (without any featured jobs yet). Test out the waters and see if we get anyone contacting us.

Reply:
Agreed. You've got the structure in place for jobs when they come in.

I'd say create the default, see what traffic - if any - we get towards that feature.

Back-end can look at the JSON in this PR as an example to prepare for, and the ticket can be ✅